test/extract_metapaths_test.py

```diff
@@ -0,0 +1,72 @@
+import pathlib
+
+import pytest
+
+import hetio.readwrite
+
+
+def get_hetionet_metagraph():
+    """
+    Return the Hetionet v1.0 metagraph
+    """
+    directory = pathlib.Path(__file__).parent.absolute()
+    path = directory.joinpath('data/hetionet-v1.0-metagraph.json')
+    return hetio.readwrite.read_metagraph(path)
+
+
+def test_extract_metapaths():
+    """
+    Test metapath extraction on the Hetionet v1.0 metagraph.
+    """
+    metagraph = get_hetionet_metagraph()
+
+    # Test max_length=0
+    metapaths = metagraph.extract_metapaths('Compound', 'Disease', max_length=0)
+    assert metapaths == []
+
+    # Test source-target-length combination where no metapaths exist
+    metapaths = metagraph.extract_metapaths('Symptom', 'Pathway', max_length=2)
+    assert metapaths == []
+
+    # Test max_length=4: the metapaths used in Project Rephetio (note that
+    # Project Rephetio excluded length 1 metapaths) resulting in 1206 metapaths.
```

Comment:
Could we maybe say for clarity, `1206 + 2` metapaths? I understand that Rephetio had all but length 1, so it was 1208 - 2 = 1206 but it seems like a typo on first glance.

Reply:
Ah there's a typo in that the close parenthesis should be at the end. Would that fix your confusion? Are you saying line 32 should do `1206 + 2`?

Reply:
Moving the closing parenthesis would clear things up